src/components/Iframe.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from "react";
+import Requests from "./Requests";
+import "../styles/iframe.css";
+
+export default class Iframe extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      game: {}
```

Comment:
should add an error path if the game doesn't exists / couldn't be loaded from BE